salt/pelican/frontend/files/credentials.json

```
@@ -1 +1 @@
-{"token": "{{ pillar.google.token }}", "refresh_token": "{{ pillar.google.refresh_token }}", "token_uri": "https://oauth2.googleapis.com/token", "client_id": "{{ pillar.google.client_id }}", "client_secret": "{{ pillar.google.client_secret }}", "scopes": ["https://www.googleapis.com/auth/documents", "https://www.googleapis.com/auth/drive"]}
+{"type": "service_account", "project_id": "{{ pillar.google.project_id }}", "private_key_id": "{{ pillar.google.private_key_id }}", "private_key": "{{ pillar.google.private_key }}", "client_email": "{{ pillar.google.client_email }}", "client_id": "{{ pillar.google.client_id }}", "auth_uri": "https://accounts.google.com/o/oauth2/auth", "token_uri": "https://oauth2.googleapis.com/token", "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs", "client_x509_cert_url": "{{ pillar.google.client_x509_cert_url }}"}
```

I think the `urlencode` filter will work, to reduce one variable (https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.urlencode).

Also, we can make this more readable than the original version.

```suggestion
{
  "type": "service_account",
  "project_id": "{{ pillar.google.project_id }}",
  "private_key_id": "{{ pillar.google.private_key_id }}",
  "private_key": "{{ pillar.google.private_key }}",
  "client_email": "{{ pillar.google.client_email }}",
  "client_id": "{{ pillar.google.client_id }}",
  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
  "token_uri": "https://oauth2.googleapis.com/token",
  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/{{ pillar.google.client_email | urlencode }}"
}
```